### ISSUE TITLE
Allow users to hide display names

### ISSUE THREAD

**Issue by rixx** [mastodon/mastodon#8479]
Some people choose long/emoji overflowing display names, which isn't great accessibility wise and also plain annoying to some. Especially if people change their name frequently, it's hard to remember who that person is.

An option to hide display names and just show real user names would help both people with screenreaders and people who don't like display names.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate. It is somewhat related to #5657

**Comment by 153** [comment]
what about a forced anon mode too 

where everyone's display names and usernames are hidden

**Comment by rixx** [comment]
@153 While this is an interesting idea for sure, it would be better suited to have a separate feature request issue.

**Comment by rixx** [comment]
Closing due to lack of response.
